### PR TITLE
Fix/condo/sberdoma 1035/server side xlsx for properties export

### PR DESCRIPTION
**Pull request by dkoviazin**
Server-side export properties to xlsx
[Экспорт домов в эксель.xlsx](https://github.com/open-condo-software/condo/files/7007675/default.xlsx)

### CODE REVIEW

File: apps/condo/domains/property/constants.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/**
+ * Generated by `createservice property.ExportPropertiesToExcelService --type queries`
+ */
+
+module.exports = {
```
Comment (SavelevMatthew) [inline apps/condo/domains/property/constants.js:5]: Seems like this file is unnecessary or should we keep empty auto-generated files?

File: apps/condo/domains/property/schema/ExportPropertiesToExcelService.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    ],
+    
+    queries: [
+        {
+            access: access.canExportPropertiesToExcel,
+            schema: 'executeExportPropertiesToExcel (data: ExportPropertiesToExcelInput!): ExportPropertiesToExcelOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                const { where, sortBy } = args.data
+                let skip = 0
+                let maxCount = 1000
```
Comment (SavelevMatthew) [inline apps/condo/domains/property/schema/ExportPropertiesToExcelService.js:35]: Is this artificial limit to export? I guess you  should move it to consts like `CHUNK_SIZE` and name something like `MAX_QUERIES_COUNT`

File: apps/condo/domains/ticket/constants/errors.js [hunk 3]
```diff
@@ -1,6 +1,4 @@
 
-const EMPTY_DATA_EXPORT_ERROR = '[export.empty.data'
```
Comment (SavelevMatthew) [inline apps/condo/domains/ticket/constants/errors.js:2]: This is empty human-generated file. I don't think we should keep it

**Review by SavelevMatthew — APPROVED** [review]
LGTM 👍 

Don't forget to clean redundant files 👀

File: apps/condo/domains/property/constants.js (hunk 1 above)
Comment (dkoviazin) [reply apps/condo/domains/property/constants.js:5]: Ok

File: apps/condo/domains/ticket/constants/errors.js (hunk 3 above)
Comment (dkoviazin) [reply apps/condo/domains/ticket/constants/errors.js:2]: done

File: apps/condo/domains/property/schema/ExportPropertiesToExcelService.js (hunk 2 above)
Comment (dkoviazin) [reply apps/condo/domains/property/schema/ExportPropertiesToExcelService.js:35]: renamed to iterationsLimit

File: apps/condo/domains/property/schema/ExportPropertiesToExcelService.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        {
+            access: true,
+            type: 'type ExportPropertiesToExcelOutput { status: String!, linkToFile: String!  }',
+        },
+    ],
+    
+    queries: [
+        {
+            access: access.canExportPropertiesToExcel,
+            schema: 'executeExportPropertiesToExcel (data: ExportPropertiesToExcelInput!): ExportPropertiesToExcelOutput',
```
Comment (pahaz) [inline apps/condo/domains/property/schema/ExportPropertiesToExcelService.js:31]: Just exportPropertiesToExcel

Comment (dkoviazin) [reply apps/condo/domains/property/schema/ExportPropertiesToExcelService.js]: renamed (it was autogenerated name)

**Review by Dimitreee — APPROVED** [review]

**Pull request merged**